src/collection/forEachRight/index.test.js

```diff
@@ -0,0 +1,24 @@
+const { forEachRight } = require('./index')
+
+it('should iterate each element of array.', () => {
+  const arr = [1, 2, 3]
+  const resultArr = []
+
+  forEachRight(arr, value => {
+    resultArr.push(value)
+  })
+  expect(resultArr).toEqual(arr)
```

Comment:
add this line here:
```expect(arr).toEqual([1,2,3])```
It will fail. Please change forEachRight function to make the test green.